README.md

```diff
@@ -330,40 +332,51 @@ export class CasbinResValModifierProvider
     return `${res}`;
   }
 }
-
 ```
+
 - Implement the **casbin enforcer config provider** . Provide the casbin model path. In case 1 of using [default casbin format policy!](https://casbin.org/docs/en/how-it-works), provide the casbin policy path. In other case of creating dynamic policy, provide the array of Resource-Permission objects for a given user, based on business logic.
 
 ```ts
 import {Provider} from '@loopback/context';
-import {CasbinConfig, CasbinEnforcerConfigGetterFn, IAuthUserWithPermissions} from 'loopback4-authorization';
+import {
+  CasbinConfig,
+  CasbinEnforcerConfigGetterFn,
+  IAuthUserWithPermissions,
+} from 'loopback4-authorization';
 import * as path from 'path';
 
 export class CasbinEnforcerConfigProvider
   implements Provider<CasbinEnforcerConfigGetterFn> {
   constructor() {}
 
   value(): CasbinEnforcerConfigGetterFn {
-    return (authUser: IAuthUserWithPermissions, resource: string, isCasbinPolicy?: boolean) =>
-      this.action(authUser, resource, isCasbinPolicy);
+    return (
+      authUser: IAuthUserWithPermissions,
+      resource: string,
+      isCasbinPolicy?: boolean,
+    ) => this.action(authUser, resource, isCasbinPolicy);
   }
 
-  async action(authUser: IAuthUserWithPermissions, resource: string, isCasbinPolicy?: boolean): Promise<CasbinConfig> {
-    const model = path.resolve(
-      __dirname,
-      './../../fixtures/casbin/model.conf',
-    );
+  async action(
+    authUser: IAuthUserWithPermissions,
+    resource: string,
+    isCasbinPolicy?: boolean,
+  ): Promise<CasbinConfig> {
+    const model = path.resolve(__dirname, './../../fixtures/casbin/model.conf');
```

Comment:
why path from fixtures ?

Reply:
@samarpan-b Fixtures folder is in the application consuming the extension. We can name it differently also, as you suggest.

Reply:
hmm. we should ask this data from consumer. path or actual content, whatever. can we do this easily ?

Reply:
Done. Updated PR with code to initialise model from file, code or a string. Also updated readme for that.